### PR TITLE
add group norm benchmark

### PR DESCRIPTION
**Pull request by liqiangxl**
add group norm benchmark, NCHW -->NG(C/G)HW
**input sizes:** `generate_input_sizes(dims=4)`
**n_groups:** 
```
    # start num_groups from 1 and increase to 32 at max
    num_groups = 1
    while num_groups * 2 <= 32 and C % (num_groups * 2) == 0:
        num_groups *= 2
```
TODO:
add channel last

### CODE REVIEW

File: benchmarks/python/test_groupnorm_fwd.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    clear_cuda_cache()
+
+    N, C, H, W = size
+    x = torch.randn(size, device="cuda", dtype=dtype)
+    weight = torch.randn(C, device="cuda", dtype=dtype)
+    bias = torch.randn(C, device="cuda", dtype=dtype)
+    # start num_groups from 1 and increase to 32 at max
+    num_groups = 1
+    while num_groups * 2 <= 32 and C % (num_groups * 2) == 0:
+        num_groups *= 2
```
Comment (Priya2698) [inline benchmarks/python/test_groupnorm_fwd.py:108]: Can you add a comment here, explaining the reasoning behind this?

File: benchmarks/python/test_groupnorm_fwd.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+
+def groupnorm_fwd(inputs: list):  # [in_tensor, weights, bias, n_groups]
+    return torch.nn.functional.group_norm(
+        inputs[0], num_groups=inputs[3], weight=inputs[1], bias=inputs[2], eps=1e-05
+    )
+
+
+@pytest.mark.parametrize("size", generate_input_sizes(dims=4))
+@pytest.mark.parametrize("dtype", FLOAT_DTYPES)
```
Comment (Priya2698) [inline benchmarks/python/test_groupnorm_fwd.py:90]: Is `channels_last` going to be part of this PR or will be added later?

File: benchmarks/python/test_groupnorm_fwd.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+def test_groupnorm_fwd_nvf_benchmark(
```
Comment (Priya2698) [inline benchmarks/python/test_groupnorm_fwd.py:91]: Should we add a marker here for the scheduler too?

Comment (liqiangxl) [reply benchmarks/python/test_groupnorm_fwd.py:92]: added

File: benchmarks/python/test_groupnorm_fwd.py (hunk 2 above)
Comment (liqiangxl) [reply benchmarks/python/test_groupnorm_fwd.py:90]: in a different PR.

**Review by Priya2698 — APPROVED** [review]
LGTM.
Please add a comment on how the number of groups is chosen for the fusions as well.
